#158487956 Integrate TravisCI

Stylll opened the pull request:
#### What does this PR do?
Integrates TravisCI and adds readme badge.

#### Description of Task to be completed?
- Create travis config file to integrate app with TravisCI
- Add build badge to readme file

#### How should this be manually tested?
- Check for the .travis.yml file in the root of the folder.
- Check the readme file to view the TravisCI badge.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#158487956

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

Stylll committed:
chore(travis): integrate travis ci
- automate unit tests with travis ci
- add travis build badge to readme file
[Finishes #158487956]

Stylll committed:
chore(travis): integrate travis ci
- automate unit tests with travis ci
- add travis build badge to readme file
[Finishes #158487956]

Stylll committed:
chore(travis): integrate travis with readme badge
- resolve merge conflict
- configure env paths
[Fixes #130635935]

VictoriaOffoma commented on test/article/model.js:
```diff
@@ -1,58 +1,58 @@
+//             done();
+//         });
+//     });
+// });
```
Please add newline

Stylll replied:
Newline added

VictoriaOffoma commented on config/config.js:
```diff
@@ -4,4 +4,4 @@ var _ = require('underscore');
     require(__dirname + '/../config/env/all.js'),
-    require(__dirname + '/../config/env/' + process.env.NODE_ENV + '.json') || {});
\ No newline at end of file
+    require(__dirname + '/../config/env/' + process.env.NODE_ENV + '.js') || {});
```
Please add a newline.

Stylll replied:
Newline added.

Stylll committed:
chore(travis): integrate travis with readme badge
- add new line to model.js and config.js
[Fixes #130635935]

VictoriaOffoma commented on config/config.js:
```diff
@@ -4,4 +4,5 @@ var _ = require('underscore');
-    require(__dirname + '/../config/env/' + process.env.NODE_ENV + '.js') || {});
\ No newline at end of file
+    require(__dirname + '/../config/env/' + process.env.NODE_ENV + '.js') || {});
+    
```
hey, one escaped you.

Stylll replied:
Line 8 is empty. It can serve as a new line right ?. Is there something I'm missing ?.

Stylll committed:
Merge branch 'staging' into chore/158487956/travis-ci-setup

Stylll committed:
chore(travis): integrate travis with readme badge
- add new line to config.js file
- remove addon from travis yml file
[Fixes #130635935]
